src/main/java/javaslang/collection/Stream.java

```diff
@@ -1050,6 +1051,19 @@ default int length() {
     }
 
     @Override
+    default <U extends Comparable<? super U>> Stream<T> sortBy(Function<? super T, ? extends U> mapper) {
+        return sortBy(U::compareTo, mapper);
```

Comment:
Yes, figured it out by accident [while implementing sorted collections](https://gist.github.com/danieldietrich/d15081d84fce08e30153). Really sweet!